.github/workflows/preview-docs.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Install dependencies
         run: |
-          pip install requests
+          pip install python3-requests
```

Comment:
For fixing https://github.com/mlflow/mlflow/actions/runs/11321199491/job/31480737038?pr=13406